src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -275,14 +280,28 @@ private String createErrorResponse(final String message, final ArrayList<String>
         return errorResponse.toString();
     }
 
-    private void outputValidate(EiffelEventType eiffelType, String jsonStringInput) throws EiffelValidationException {
+    private JsonObject outputValidate(EiffelEventType eiffelType, String jsonStringInput, Boolean lenientValidation) throws EiffelValidationException {
         EiffelValidator validator = EiffelOutputValidatorFactory.getEiffelValidator(eiffelType);
         JsonObject jsonObject = new JsonParser().parse(jsonStringInput).getAsJsonObject();
-        validator.validate(jsonObject);
+        JsonObject validate = validator.validate(jsonObject, lenientValidation);
         // custom validations on an event which is not covered in schema.
-        validator.customValidation(jsonObject);
+        validator.customValidation(validate);
+        return validate;
     }
 
+    @Override
+    public ValidationResult validateMsg(String msgType, JsonObject jsonvalidateMessage, Boolean lenientValidation) {
```

Comment:
The addition of the lenientValidation parameter to a public method isn't a backwards compatible change.

Reply:
added another method